components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/utils/GitpodLogger.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.utils
+
+import com.intellij.openapi.diagnostic.logger
+
+class GitpodLogger {
+    companion object {
+        private val logger = logger<GitpodLogger>()
+
+        /**
+         * When running launch-dev-server.sh, only logs with levels higher than Info (e.g. Warn, Error)
+         * are displayed on terminal, that's why we should log Info as Warn during development.
+         */
+        private val shouldLogInfoAsWarn = System.getenv("JB_DEV").toBoolean()
+
+        private const val messagePrefix = "[Gitpod Backend Plugin]"
```

Comment:
Can we align it with `GItpodManager` that output is the same. `gitpod: ` is enough?

Reply:
I agree, then I'd like us to make the string `gitpod: ` a constant that we can import when logging, just to avoid typos and string repetition.

I've updated the code removing the GitpodLogger class.